Libplanet.Tests/Blockchain/BlockChainTest.MineBlock.cs

```diff
@@ -608,6 +608,25 @@ public async Task IgnoreLowerNonceTxsAndMine()
             Assert.Contains(txsB[3], b2.Transactions);
         }
 
+        [Fact]
+        public async Task IgnoreDuplicatedNonceTxs()
+        {
+            var privateKey = new PrivateKey();
+            var address = privateKey.ToAddress();
+            var txs = Enumerable.Range(0, 3)
+                .Select(_ => _fx.MakeTransaction(
+                    nonce: 0,
+                    privateKey: privateKey,
+                    timestamp: DateTimeOffset.Now))
+                .ToArray();
+            StageTransactions(txs);
+            Block<DumbAction> b = await _blockChain.MineBlock(address, append: false);
+            _blockChain.Append(b);
+
+            Assert.Single(b.Transactions);
+            Assert.Equal(txs[0], b.Transactions.Single());
```

Comment:
Do we want the selection of the "first" one to be included in the spec?

Reply:
I haven't thought deeply about it... but it seems better to come out first in the order you put it than to come out randomly. (This is how it works now)

Reply:
Eh, I guess we can write more tests in the future if we decide to implement pluggable selection strategy.